modules/promotion/src/PromotionStorage.php

```diff
@@ -104,4 +104,61 @@ public function loadAvailable(OrderTypeInterface $order_type, StoreInterface $st
     return $promotions;
   }
 
+  /**
+   * Return active promotions that have passed their end date.
```

Comment:
Document blocks should only be on the interface, implementers just do `{@inheritdoc}`